datafusion/common/src/scalar/mod.rs

```diff
@@ -231,6 +233,10 @@ pub enum ScalarValue {
     Float32(Option<f32>),
     /// 64bit float
     Float64(Option<f64>),
+    /// 32bit decimal, using the i32 to represent the decimal, precision scale
+    Decimal32(Option<i32>, u8, i8),
```

Comment:
I think 🤔  this is technically a breaking API change I think because `ScalarValue` is a public enum and not parsed `non_exhastive`... So we can't backport it to the branch-50 without breaking the API

Reply:
That makes perfect sense, should've noticed it myself.
I can wait until November, sorry about causing so much noise pushing for this release, at least we'll get the docs fix in.